src/js/components/Calendar/stories/CustomDay.js

```diff
@@ -60,14 +53,15 @@ export const CustomDayCalendar = () => {
         <Calendar onSelect={onSelect} date={selectedDay} fill>
           {({ day, isSelected }) => (
             <Box background={isSelected ? 'light-3' : 'white'} border fill>
-              <Box pad="medium" align="center" justify="center" fill>
+              <Box pad="small" align="center" justify="center" fill>
```

Comment:
Why is the pad changing from medium to small here?

Reply:
When I opened the storybook I saw two differents size calendars, by the title I assume the only difference should be the onSelect set or not, so I changed to keep both calendars with the same height.